site/docs/v1/tech/oauth/index.adoc

```diff
@@ -84,6 +91,56 @@ The OAuth grants enabled for this Application. When creating a new Application,
 [field]#Device Verification URL# [optional]#Optional#::
 The URL to direct the end-user to for the user-interaction portion of the Device Authorization Grant.  This field is required if [field]#Device# is enabled in the OAuth `Enabled grants` for this Application.
 
+== Configure Entities
+
+The Client Credentials grant takes place between two Entities. You can learn more about link:/docs/v1/tech/core-concepts/entities/[Entities] in the Core Concepts section.
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+There are two entities which take part in a Client Credentials grant in FusionAuth: the recipient Entity and the target Entity. 
+
+Imagine you have a todo API which lets you create, read, update, and delete todos. You also have an email API, which lets you send emails. You can represent these both in FusionAuth as Entities.
+
+When building functionality to allow the todo API send email reminders, grant permissions on the email API (the target Entity) to the todo API (the recipient Entity). To set up this relationship:
+
+* Create an API entity type with the following permissions: `execute` and `configure`.
+* Create a todo API entity
+* Create an email API entity
+* Grant the todo API `execute` permissions on the email API
+
+The todo API is the recipient Entity, because it receives the permissions to call the email API. The email API is the target Entity, because it will process the token. The email API is the system to which access is controlled. 
+
+The set up happens once and then the todo API can perform the client credentials grant any time it needs to call the email API. It will get a token at the end of a successful grant and can present that to the email API. 
+
+You may set up the entities via the FusionAuth API or the administrative user interface. You can specify the [field]#Client Id# and [field]#Client Secret# if desired.
+
+image::oauth/add-entity-client-credentials.png[Setting up an Entity,width=1200,role=bottom-cropped]
+
+Grants can be made between entities only via the API.
```

Comment:
This can be done in the admin UI?

![Screen Shot 2021-05-05 at 2 03 47 PM](https://user-images.githubusercontent.com/1679571/117202154-bb19ae00-adaa-11eb-8766-70c78781c673.png)

Reply:
Sick! I missed that! Will take a screenshot and add that in.